### DISCUSSION TITLE
Make the video transcripts searchable?

### DISCUSSION THREAD

**Discussion by ghost** [pointfreeco/pointfreeco#842] [General]
Hi @mbrandonw and @stephencelis,

Would you consider making the text of the video transcripts globally searchable? (Globally as in across videos)

Often times I have a vague recollection of something you have talked about in one of the videos but I can't recall which one and it would be nice if we could search the entire text.

Case in point. I'm trying to create a box type similar to `AnyEquatable` and I remember that you explained how to do it using open existentials. Of course, I can find `AnyEquatable` in the codebase for The Composable Architecture but what I'm looking for is the _understanding_ that you provided in the video, which I can't reconstruct off the top of my head. Unfortunately, I can't recall which video that was in.

Many thanks in advance.

**Comment by TC72** [comment]
Even better make it AI sexy by creating a VectorDB and hook it up to a Chat GPT retrieval plugin.  👍
